src/displayapp/screens/ApplicationList.h

```diff
@@ -33,26 +33,23 @@ namespace Pinetime {
         Pinetime::Controllers::Ble& bleController;
         Controllers::DateTime& dateTimeController;
 
-        static constexpr int appsPerScreen = 6;
-
-        // Increment this when more space is needed
-        static constexpr int nScreens = 2;
-
-        static constexpr std::array<Tile::Applications, appsPerScreen * nScreens> applications {{
+        static constexpr Tile::Applications applications[] {
           {Symbols::stopWatch, Apps::StopWatch},
           {Symbols::clock, Apps::Alarm},
           {Symbols::hourGlass, Apps::Timer},
           {Symbols::shoe, Apps::Steps},
           {Symbols::heartBeat, Apps::HeartRate},
           {Symbols::music, Apps::Music},
-
```

Comment:
I personally think that it's better for readability to leave the empty lines between screens.

Reply:
The purpose of this PR is to create separation between the list of applications and the created screens. Otherwise there should just be an array per screen.